libevmasm/Assembly.cpp

```diff
@@ -264,7 +264,7 @@ Json::Value Assembly::assemblyJSON(StringMap const& _sourceCodes) const
 					createJsonValue("PUSH [ErrorTag]", i.location().start, i.location().end, ""));
 			else
 				collection.append(
-					createJsonValue("PUSH [tag]", i.location().start, i.location().end, string(i.data())));
```

Comment:
I think your version creates a hex string while it previously created a decimal string, so I think this should either use `i.data().str()` or `to_string(i.data())` whichever works.

Reply:
This is also showing there are no tests against this piece of code.